packages/ui/src/components/histories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import type { GetProps } from 'tamagui';
+import { ListItem, YGroup, ScrollView, Paragraph } from 'tamagui';
+import { ChevronRight, History, Image } from '@tamagui/lucide-icons';
+import { ListItem as TamaguiListItem } from 'tamagui';
+import { Title1 } from './typographies/title1';
+import { withVariants } from '../utils/with-variants';
+import { View } from 'react-native';
+
+const formatDate = (dateString: string) => {
+  const date = new Date(dateString);
+  return date.toLocaleDateString('fr-FR', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+};
+
+interface Journey {
+  name: string;
+}
+
+interface ItemProps {
+  id?: number;
+  createdAt: string;
+  updatedAt?: string;
+  userId?: number;
+  journey_id?: number;
+  ended_at?: number;
+  journey?: Journey;
+
+  action?: {
+    onClick: () => void;
+    text: string;
+  };
+}
+
+interface HistoryProps {
+  histories: ItemProps[];
+}
+
+export const Histories = withVariants<
```

Comment:
There is no point to put `Histories` component in the `ui` package. `ui` package is useful to share the components used in dashboard and mobile app (like buttons, inputs, cards, etc.). This component is very specific to mobile so it shouldn't be there, neither use Tamagui. Please move this in mobile folder, and remove usage of Tamagui (instead use the `Box` component from `ui` and built-in React Native components)